glide.yaml

```diff
@@ -72,3 +72,14 @@ import:
   subpackages:
   - compute/metadata
   - internal
+- package: k8s.io/kubernetes
+  version: e4a1ca7105fa9a3075fbf13a3286f8dddd59a658
```

Comment:
Use glide 0.12.x, it's faster/better. Also, what command are you running exactly?

If on glide 0.12.x+ it should be roughly:

```
glide update --strip-vendor --skip-test
```

Also, can you separate out changing `glide.yaml` and `glide.lock` from the changes in vendor? It's impossible to see what changed in the manifest without checking out locally. (But I will so I can look).